addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/handler/NestStructureHandler.java

```diff
@@ -0,0 +1,95 @@
+package org.openhab.binding.nest.handler;
+
+import static org.openhab.binding.nest.NestBindingConstants.*;
+
+import java.util.Calendar;
+import java.util.TimeZone;
+
+import org.eclipse.smarthome.core.library.types.DateTimeType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.Channel;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.nest.NestBindingConstants;
+import org.openhab.binding.nest.internal.NestUpdateRequest;
+import org.openhab.binding.nest.internal.data.Structure;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Deals with the structures on the nest api, turning them into a thing in openhab. */
+public class NestStructureHandler extends BaseNestHandler {
```

Comment:
Please add a JavaDoc with `@author` tag containing a real name of the author. See [Code Style](http://docs.openhab.org/developers/development/guidelines.html#a-code-style).